VCFv4.3.tex

```diff
@@ -43,6 +43,11 @@ \section{The VCF specification}
 In order to ensure interoperability across platforms, VCF compliant implementations must support
 both LF (\texttt{\textbackslash n}) and CR+LF (\texttt{\textbackslash r\textbackslash n}) newline conventions.  
 
+Unless explicitly specified elsewhere, all fields are encoded using 7-bit
```

Comment:
Aye, I didn't read it fully before adding that.  My error - I'll drop it and force push a new update with SAM only.